tests/test_rocket.py

```diff
@@ -370,13 +370,7 @@ def test_evaluate_static_margin_assert_cp_equals_cm(kg, m, dimensionless_rocket)
 
 @pytest.mark.parametrize(
     "k, type",
-    (
-        [2 / 3, "conical"],
-        [0.466, "ogive"],
-        [0.563, "lvhaack"],
-        [0.5, "default"],
-        [0.5, "not a mapped string, to show default case"],
-    ),
+    ([2 / 3, "conical"], [0.46469957130675876, "ogive"], [0.563, "lvhaack"]),
```

Comment:
Regarding the `0.46469957130675876` value: 
I did the bureaucracy so the tests could pass, but I'd love to have more reasons on why did the "ogive" format had its k value changed.

Regarding the others that were removed, it is expected since the now would be caught in exception error by the kind setter in the NoseCone class.

Reply:
I didn't understand the problem here.

Reply:
> I didn't understand the problem here.
> 
> Could you be more specific please?

I think you already solved the question by answering another comment below. What I was wondering was regarding the value change of the variable k. But now it is clear that it comes from the new reference that you pointed out.